stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -167,93 +168,98 @@ public View getViewAndBoundsForHighlighting(View element, Rect bounds) {
   }
 
   @Override
-  protected void onGetStyles(View element, StyleAccumulator styles) {
-    List<ViewCSSProperty> properties = getViewProperties();
-    for (int i = 0, size = properties.size(); i < size; i++) {
-      ViewCSSProperty property = properties.get(i);
-      try {
-        getStyleFromValue(
-            VIEW_SELECTOR_NAME,
-            element,
-            property.getCSSName(),
-            property.getValue(element),
-            property.getAnnotation(),
-            styles);
-      } catch (Exception e) {
-        if (e instanceof IllegalAccessException || e instanceof InvocationTargetException) {
-          LogUtil.e(e, "failed to get style property " + property.getCSSName() +
-                  " of element= " + element.toString());
-        } else {
-          throw ExceptionUtil.propagate(e);
+  protected void onGetStyleRules(View element, StyleRuleAccumulator accumulator) {
+    accumulator.store(VIEW_STYLE_RULE_NAME, true);
+    accumulator.store(ACCESSIBILITY_STYLE_RULE_NAME, false);
+  }
+
+  @Override
+  protected void onGetStyles(View element, String ruleName, StyleAccumulator accumulator) {
+    if (VIEW_STYLE_RULE_NAME.equals(ruleName)) {
+      List<ViewCSSProperty> properties = getViewProperties();
+      for (int i = 0, size = properties.size(); i < size; i++) {
+        ViewCSSProperty property = properties.get(i);
+        try {
+          getStyleFromValue(
+                  element,
```

Comment:
nit: I'm seeing this all over the diff: why 8-space continuation indents? should be 4 spaces